presentations/posters/khider_LinkedEarth_2024/info.json

```
@@ -0,0 +1,44 @@
+{
+    "authors": [
+        {
+            "name": "Deborah Khider",
+            "affiliation": "USC Information Sciences Institute",
+            "orcid": "0000-0001-7501-8430"
+        },
+        {
+            "name": "Varun Ratnakar",
+            "affiliation": "USC Information Sciences Institute",
+            "orcid": "0000-0001-9381-705X"
+        },
+        {
+            "name": "Julien Emile-Geay",
+            "affiliation": "University of Southern California",
+            "orcid": "0000-0001-5920-4751"
+        },
+        {
+            "name": "Kim Pevey",
+            "affiliation": "Quansight"
```

```suggestion
            "affiliation": "Quansight",
            "orcid": ""
```
Hi! Could you please use an empty string for author orcid if it cannot be provided?